Admin/Manager/DoctrineMongoDBManager.php

```diff
@@ -19,6 +19,8 @@
 /**
  * this method overwrite the default AdminModelManager to call
  * the custom methods from the dedicated media manager.
+ *
+ * @deprecated
```

Comment:
Can you please leave a comment here?

``@deprecated since XXX (to be removed in 3.0)``